server/game/cards/15-DotE/Bribery.js

```diff
@@ -11,33 +11,25 @@ class Bribery extends DrawCard {
                     (!context.xValue || card.getPrintedCost() <= context.xValue) &&
                     (
                         card.isMatch({ kneeled: false }) && card.allowGameAction('kneel') ||
-                        card.isMatch({ trait: ['Ally', 'Mercenary'] })
+                        card.isMatch({ trait: ['Ally', 'Mercenary'] }) && card.allowGameAction('takeControl')
                     )
                 )
             },
             handler: context => {
-                const buttons = [];
-
-                this.context = context;
-                if(!context.target.kneeled && context.target.allowGameAction('kneel')) {
-                    buttons.push({ text: 'Kneel', method: 'kneelCharacter' });
-                }
-
-                if(context.target.hasTrait('Ally') || context.target.hasTrait('Mercenary')) {
-                    buttons.push({ text: 'Take control', method: 'takeControl' });
-                }
-
-                if(buttons.length > 1) {
-                    this.game.promptWithMenu(context.player, this, {
-                        activePrompt: {
-                            menuTitle: 'Take control of character?',
-                            buttons: buttons
+                this.game.resolveGameAction(
+                    GameActions.ifCondition({
+                        condition: context => context.target.isMatch({ trait: ['Ally', 'Mercenary'] }),
+                        thenAction: {
+                            message: '{player} plays {source} to take control of {target}',
+                            gameAction: GameActions.takeControl(context => ({ player: context.player, card: context.target }))
                         },
-                        source: this
-                    });
-                } else {
-                    this[buttons[0].method](context.player);
-                }
+                        elseAction: {
+                            message: '{player} players {source} to kneel {target}',
+                            gameAction: GameActions.kneelCard(context => ({ card: context.target }))
+                        }
+                    }),
+                    context

```

Comment:
I'm not sure how much it matters (since this is the original version of the card, not the redesign), but this now forces you to take control if you can, whereas the card text gives you a choice ('you may take control instead')

If we don't care, then this should be good to merge.

Reply:
@ystros Should be fixed now in 1e58741. There might be some small cases where a kneeling Ally/Mercenary will still give you a choice to kneel it when it shouldn't, but I think we can leave it (as that seems more like something we should fix with `GameActions.choose` rather than this card specifically. Maybe an option within choose to eliminate options if the condition or allow isn't met (where currently, if 1 is available, all should be available).

Can merge this if youre happy! :)